src/WinRT.Runtime2/InteropServices/ObjectReference/WindowsRuntimeObjectReference.Factory.cs

```diff
@@ -17,39 +17,38 @@ public unsafe partial class WindowsRuntimeObjectReference
     /// <returns>The <see cref="WindowsRuntimeObjectReference"/> holding onto the <paramref name="thisPtr"/> pointer.</returns>
     /// <remarks>
     /// <para>
-    /// This method will increment the reference count for <paramref name="thisPtr"/>. Additionally, it assumes that the input COM object
-    /// already points to the interface represented by <paramref name="iid"/>. It is responsibility of the caller to respect this invariant.
+    /// This method will perform a <c>QueryInterface</c> call on <paramref name="thisPtr"/> to retrieve the requested interface pointer.
     /// </para>
     /// <para>
     /// The resulting <see cref="WindowsRuntimeObjectReference"/> is <see langword="null"/> if <paramref name="thisPtr"/> is <see langword="null"/>.
     /// </para>
     /// </remarks>
-    public static WindowsRuntimeObjectReference? Create(void* thisPtr, in Guid iid)
+    /// <exception cref="Exception">Thrown if the <c>QueryInterface</c> operation fails.</exception>
+    internal static WindowsRuntimeObjectReference? Create(void* thisPtr, in Guid iid)
```

Comment:
Given we are switching which does the QI, we will need to go through all uses and make sure the assumptions they called it with holds.